blog/secadvisor.md

```diff
@@ -0,0 +1,290 @@
+In the future, we plan to have additional pipelines performing different kinds of operations.
+
+## Deploy Skydive
+
+You can either download a statically linked version of Skydive or you can build it on your own.
+
+To download a pre-compiled statically linked version of Skydive, run the following command.
+
+```
+curl -Lo skydive https://github.com/skydive-project/skydive-binaries/raw/jenkins-builds/skydive-latest && chmod +x skydive && sudo mv skydive /usr/local/skydive/
+```
+
+To build your own version of Skydive, follow the instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the Skydive code.
+Enter the Skydive root directory and build the code.
+
+```
+cd $GOPATH/src/github.com/skydive-project/skydive
+make
+cp etc/skydive.yml.default /etc/skydive/skydive.yml
+// adjust settings in skydive.yml, if desired
+```
+
+The skydive binary is created in $GOPATH/bin/skydive.
+
+### All-in-one version
+
+```
+sudo skydive allineone -c /etc/skydive/skydive.yml
+```
+
+### Multi-node Deployment
+
+Alternatively, run a Skydive analyzer on one host and a Skydive agent on each of your hosts.
+
+On one machine:
+
+```
+sudo skydive analyzer -c /etc/skydive/skydive.yml
+```
+On each machine:
+
+```
+sudo skydive agent -c /etc/skydive/skydive.yml
+```
+
+Be sure to set the field "analyzers" in the skydive.yml to point to the analyzer.
+
+
+## Create Security Advisor (Pipeline) Configuration File
+
+Start from the secadvisor.yml.default in the secadvisor directory.
+
+
+```
+cp secadvisor.yml.default /etc/skydive/secadvisor.yml
+```
+
+The secadvisor.yml is used as a parameter when running secadvisor from the command line.
+
+The default secadvisor.yml has the following fields:
+
+```
+host_id: ""
+analyzers:
+  127.0.0.1:8082
+pipeline:
+  analyzer:
+    subscriber_url: ws://127.0.0.1:8082/ws/subscriber/flow
+    subscriber_username:
+    subscriber_password:
+  classify:
+    # cluster_net_masks:
+      # - 10.0.0.0/8
+      # - 172.16.0.0/12
+      # - 192.168.0.0/16
+  filter:
+    # excluded_tags:
+      # - internal
+      # - other
+      # - ingress
+      # - egress
+  transform:
+    sa:
+      # exclude_started_flows: true
+  store:
+    type: s3
+    s3:
+      # -- client parames --
+      endpoint: http://127.0.0.1:9000
+      region: local
+      bucket: bucket
+      access_key: user
+      secret_key: password
+      # api_key: key
+      # iam_endpoint: https://iam.cloud.ibm.com/identity/token
+      object_prefix: logs
+      # -- bulk store params --
+      max_flows_per_object: 6000
+      max_seconds_per_object: 60
+      max_seconds_per_stream: 86400
+      max_flow_array_size: 100000
+```
+
+### Description of fields in secadvisor.yml file
+
+Entries that begin with '#' are comments and are ignored. Be sure to include/exclude the appropriate configuration parameters.
```

Comment:
I suggest using a tabular view for this section

Reply:
same here back quote